persistent/_compat.py

```diff
@@ -52,3 +53,5 @@ def _native(s):
 
     PYTHON3 = False
     PYTHON2 = True
+
+    intern = intern
```

Comment:
Is this to appease pylint/pyflakes or something?  A comment might make it clearer.